lms/djangoapps/certificates/tasks.py

```diff
@@ -43,3 +53,39 @@ def generate_certificate(self, **kwargs):
         if expected_verification_status != actual_verification_status:
             raise self.retry(kwargs=original_kwargs)
     generate_user_certificates(student=student, course_key=course_key, **kwargs)
+
+
+ACE_ROUTING_KEY = getattr(settings, 'ACE_ROUTING_KEY', None)
+
+
+class PassedCourse(MessageType):
+    pass
+
+
+@task(ignore_result=True, routing_key=ACE_ROUTING_KEY)
+def send_passing_learner_message(user_id, course_key_str):
+    try:
```

Comment:
We should remove this try-except block... helpful for devstack but harmful in prod